source/_integrations/group.markdown

```diff
@@ -58,6 +60,21 @@ Binary sensor, light, and switch groups allow you set the "All entities" option.
 - Otherwise, the group state is `off` if at least one group member is `off`.
 - Otherwise, the group state is `on`.
 
+### Climate groups
+In short, the state shown is using the order in the `HVACMode` enum (except that `off` is the least priority). E.g. when any group member entity is `heat`, the group will also be `heat`. A complete overview of how climate groups behave:
+
+- The group state is `unavailable` if all group members are `unavailable`.
+- Otherwise, the group state is `unknown` if all group members are `unknown` or `unavailable`.
+- Otherwise, the group state is `heat` if at least one group member is `heat`.
+- Otherwise, the group state is `cool` if at least one group member is `cool`.
+- Otherwise, the group state is `heat_cool` if at least one group member is `heat_cool`.
+- Otherwise, the group state is `auto` if at least one group member is `auto`.
+- Otherwise, the group state is `dry` if at least one group member is `dry`.
```

Comment:
This is odd?

So if I have a group with five climate entities, 4 of them are cooling and one is heating, it becomes heating? That sounds very unexpected.

Reply:
Yeah, it used to be that because heating would "cost" the most, so if any device is heating, you might want to know that. But, maybe it's easier to just exclude `off` entities and report the most common state. WDYT?